src/project-css/app/Console/Commands/importTable.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Helpers\CollectionHelper;
+use App\Helpers\TableHelper;
+use App\Models\Collection;
+use Illuminate\Support\Facades\Schema;
+
+class importTable extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'table:import {collectioname} {tablename} {filename}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Import File into a table and create collection if missing';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        if (Schema::hasTable($this->argument('tablename'))) { 
+            return $this->error('Table Name Already Exists.');
+        }
```

Comment:
Only returns an error string?  This seems like a poor return setup, the only time you will get a return is if the item fails or does another function return information?

Reply:
At the end of the handle() there is a confirmation line that gets returned to the console.

$this->info('File ' . $this->argument('filename') . ' Has Been Queued For Import.');